notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>For time-series based aggregation, take a look at the <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Grouper.html" target="_blank">pandas.Grouper documentation</a> -- there's a bunch of time-specific functionality that knows how to work with datetime columns directly at whatever frequency you're interested in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>